lib/main.dart

```diff
@@ -94,7 +94,9 @@ class _AppState extends ConsumerState<App> {
   // This widget is the root of the application.
   @override
   Widget build(BuildContext context) {
-    final theme = ref.read(ThemeProviders.theme);
+    final theme = ref.watch(ThemeProviders.theme);
+    final language = ref.watch(LanguageProviders.selectedLanguage);
```

Comment:
it could be interesting to have the same naming between, theme and language. So in this case: "ThemeProviders.selectedTheme".